input/docs/cs/noise_filtering.md

```diff
@@ -0,0 +1,72 @@
+title: Filtrování šumu.
+---
+>This document works with two images:
+* [Image of Norway with noise](../media/nor256_noise.png) in `A` 
+* [Image of Norway](../media/nor256.jpg) in `B`
+# Prozkoumání šumu
+``` matlab
+A = imread('nor256_noise.png');
+B = rgb2gray(imread('nor256.jpg'));
+Noise = A ( A~=B );
+imhist(Noise) %funkce zobrazí histogram.
+```
+# Gaussianová konvoluce.
```

Comment:
Proč je tu tečka a jinde ne, vždyť to nedává smysl... Pojem gausianová konvoluce je také nesmyslný... Původní anglický pojem "gauss convolution" je spíš "Konvoluce s Gaussovským filtrem", což je zřetelné z toho kódu.. A součástí přepisu do nové dokumentace jsou i takovéto korekce..